event_dispatcher.rst

```diff
@@ -322,6 +322,17 @@ its name:
 
     $ php bin/console debug:event-dispatcher kernel.exception
 
+To get everything matching "kernel" (i.e. ``kernel.exception``, ``kernel.response`` etc.) use:
```

Comment:
```suggestion
or can get everything matching "kernel" (i.e. ``kernel.exception``, ``kernel.response`` etc.):
```

Sorry for so many suggestions 🙈 💟

Reply:
Hmm. okey, you want to to match with the paragraph above.

Reply:
Yes it reads more fluently